postman/collection-dir/cybersource/Flow Testcases/Happy Cases/Scenario11-Save card payment with manual capture/Payments - Create/request.json

```
@@ -0,0 +1,104 @@
+{
+  "method": "POST",
+  "header": [
+    {
+      "key": "Content-Type",
+      "value": "application/json"
+    },
+    {
+      "key": "Accept",
+      "value": "application/json"
+    }
+  ],
+  "body": {
+    "mode": "raw",
+    "options": {
+      "raw": {
+        "language": "json"
+      }
+    },
+    "raw_json_formatted": {
+      "amount": 6540,
+      "currency": "USD",
+      "confirm": true,
+      "capture_method": "automatic",
+      "capture_on": "2022-09-10T10:11:12Z",
+      "amount_to_capture": 6540,
+      "customer_id": "adyensavecard_{{random_number}}",
+      "email": "guest@example.com",
+      "name": "John Doe",
+      "phone": "999999999",
+      "phone_country_code": "+65",
+      "description": "Its my first payment request",
+      "authentication_type": "no_three_ds",
+      "return_url": "https://google.com",
+      "setup_future_usage": "on_session",
+      "payment_method": "card",
+      "payment_method_type": "credit",
+      "payment_method_data": {
+        "card": {
+          "card_number": "371449635398431",
+          "card_exp_month": "03",
+          "card_exp_year": "2030",
+          "card_holder_name": "joseph Doe",
+          "card_cvc": "7373"
+        }
+      },
+      "billing": {
+        "address": {
+          "line1": "1467",
+          "line2": "Harrison Street",
+          "line3": "Harrison Street",
+          "city": "San Fransico",
+          "state": "California",
+          "zip": "94122",
+          "country": "US",
+          "first_name": "joseph",
+          "last_name": "Doe"
+        },
+        "phone": {
+          "number": "8056594427",
+          "country_code": "+91"
+        }
+      },
+      "shipping": {
+        "address": {
+          "line1": "1467",
+          "line2": "Harrison Street",
+          "line3": "Harrison Street",
+          "city": "San Fransico",
+          "state": "California",
+          "zip": "94122",
+          "country": "US",
+          "first_name": "joseph",
+          "last_name": "Doe"
+        },
+        "phone": {
+          "number": "8056594427",
+          "country_code": "+91"
+        }
+      },
+      "statement_descriptor_name": "joseph",
+      "statement_descriptor_suffix": "JS",
+      "metadata": {
+        "udf1": "value1",
+        "new_customer": "true",
+        "login_date": "2019-09-10T10:11:12Z"
+      },
+      "routing": {
+        "type": "single",
+        "data": "cybersource"
+      }
```

I would prefer checking with @Aprabhat19 or @vspecky on this once. I think we don't need to pass this anymore.

---

The PR for removing support for backwards compatibility has not yet been merged , so this is still fine but just to be compatible with the current type if we are passing this field can we pass it like this
```suggestion
      "routing": {
        "type": "single",
        "data": {
        "connector": "cybersource", 
        "merchant_connector_id": "{{mca_id}}"
      }
```